prow/metrics/http_test.go

```diff
@@ -56,3 +65,192 @@ func TestPowersOfTwoBetween(t *testing.T) {
 		})
 	}
 }
+
+func TestWriteHeader(t *testing.T) {
+	testcases := []struct {
+		name       string
+		statusCode int
+	}{
+		{
+			"StatusOK",
+			http.StatusOK,
+		},
+		{
+			"StatusNotFound",
+			http.StatusNotFound,
+		},
+	}
+	for _, tc := range testcases {
+		t.Run(tc.name, func(t *testing.T) {
+			rr := httptest.NewRecorder()
+			trw := &traceResponseWriter{ResponseWriter: rr, statusCode: http.StatusOK}
+			trw.WriteHeader(tc.statusCode)
+			if rr.Code != tc.statusCode {
+				t.Errorf("mismatch between expected and actual response headers: expected %s, got %s", http.StatusText(tc.statusCode), http.StatusText(rr.Code))
```

Comment:
```suggestion
				t.Errorf("mismatch between response headers: expected %s, got %s", http.StatusText(tc.statusCode), http.StatusText(rr.Code))
```

I think the expected and actual can be implied in tests, and it's reiterated by the _expected, got_ suffix.